src/main/java/Duke.java

```diff
@@ -18,28 +18,45 @@ public static void main(String[] args) {
                        underscore ;
         System.out.println(intro);
 
-        // Creates an ArrayList
-        ArrayList<String> tasks = new ArrayList<String>();
+        // Creates an ArrayList of Task objects
+        ArrayList<Task> tasks = new ArrayList<Task>();
 
         // Creates scanner object to handle input
         Scanner in = new Scanner(System.in);
         String command = in.nextLine();
         String output = "";
+        int taskNum = -1;
 
         while ( !command.equals("bye") ){
 
             if ( command.equals("list")){
```

Comment:
There should not be an empty space at the start of while and if conditions, and a spacing before the braces is required. 
Java Code Standard:
if (command.equals("list")) {